platform/src/main/java/org/stellar/anchor/platform/SepConfig.java

```diff
@@ -58,6 +54,9 @@ public FilterRegistrationBean<Sep10TokenFilter> sep10TokenFilter(
     FilterRegistrationBean<Sep10TokenFilter> registrationBean = new FilterRegistrationBean<>();
     registrationBean.setFilter(new Sep10TokenFilter(sep10Config, jwtService));
     registrationBean.addUrlPatterns("/sep12/*");
+    registrationBean.addUrlPatterns("/sep24/transaction");
+    registrationBean.addUrlPatterns("/sep24/transactions*");
+    registrationBean.addUrlPatterns("/sep24/transactions/*");
```

Comment:
❓  Do you really need both?

Reply:
Yeah... unfortunately, both of them are required for which I am also surprised.